File: HorseRacing.Web/src/interfaces/api/contracts/model/game/responses/check-player-connected-to-game/check-player-connected-to-game-response.ts
import type { BaseResponse } from '../../../../base/responses/base-response';
import type { CheckPlayerConnectedToGameResponseDto } from './check-player-connected-to-game-response-dto';

export class CheckPlayerConnectedToGameResponse implements BaseResponse<CheckPlayerConnectedToGameResponseDto> {
    Data: CheckPlayerConnectedToGameResponseDto;

    constructor(data?: CheckPlayerConnectedToGameResponseDto) {
        this.Data = data || { IsConnected: false };
    }
}

File: HorseRacing.Contracts/Models/Game/Responses/CheckPlayerConnectedToGame/CheckPlayerConnectedToGameResponse.cs
using HorseRacing.Contracts.Base.Responses;

namespace HorseRacing.Contracts.Models.Game.Responses.CheckPlayerConnectedToGame
{
    public class CheckPlayerConnectedToGameResponse : BaseResponse<CheckPlayerConnectedToGameResponseDto>
    {
        public CheckPlayerConnectedToGameResponse() : base() { }
        public CheckPlayerConnectedToGameResponse(CheckPlayerConnectedToGameResponseDto data) : base(data) { }
    }
}
